plugins/scaffolder-backend/src/scaffolder/actions/builtin/publish/gitlab.ts

```diff
@@ -149,19 +157,35 @@ export function createPublishGitlabAction(options: {
         id: number;
       };
 
+      const { id: userId } = (await client.Users.current()) as {
```

Comment:
The client resolution does only work, when you are authenticated via gitlab right? Cause we use an azure identity provider for login in backstage and a gitlab group access token to create resources in gitlab.
At the moment it uses the user_id of the gitlab group access token and sets it as owner of the repository.

Is there a way for us to set the owner the right way?